sympy/functions/elementary/trigonometric.py

```diff
@@ -14,6 +14,7 @@
 from sympy.functions.elementary.hyperbolic import (acoth, asinh, atanh, cosh,
     coth, HyperbolicFunction, sinh, tanh)
 from sympy.sets.sets import FiniteSet
+from sympy.utilities import filldedent
```

Comment:
Yeah !
I am a bit confused with whether to use `filldedent` when raising an exception or not.

Ping @aktech @hargup @asmeurer 
I have come across instances where either of them is used.
Do we have a general convention about this issue which I might be overlooking ?